addon/components/variable-number/number.ts

```diff
@@ -30,8 +29,9 @@ type Args = {
   contentDecorations?: DecorationSource;
 };
 export default class VariableNumberPluginNumberComponent extends Component<Args> {
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-call
   @localCopy('args.node.attrs.value', '') declare inputNumber: string;
+  @localCopy('args.node.attrs.writtenNumber', '')
```

Comment:
the second parameter is a default value if `writtenNumber` is undefined. If you do want to give it a default, this should be a boolean, not an empty string.

Reply:
Beside that: I'm guessing writtenNumber is a localCopy because AuToggleSwitch is a two-way bind (as it uses <Input> internally): https://github.com/appuniversum/ember-appuniversum/issues/375 (ugh). 
Just note that normally this wouldn't be needed. But for now it does :(